### DISCUSSION TITLE
Unable to find OrchardCore.Json

### DISCUSSION THREAD

**Discussion by weirdyang** [OrchardCMS/OrchardCore#16692] [Q&A] [answered]
I'm trying to implemetn a new workflowmanager for OC 1.8, looking at the source code, it needs OrchardCore.Json, but I can't seem to find it. I checked the OC source code and I see it's under OrchardCore.Abstractions, which I have already referenced:

![image](https://github.com/user-attachments/assets/e04b6000-25f1-4d0e-a318-9905a6f5f88c)
```
  <ItemGroup>
    <PackageReference Include="OrchardCore.Abstractions" Version="1.8.3" />
    <PackageReference Include="OrchardCore.ContentManagement" Version="1.8.3" />
    <PackageReference Include="OrchardCore.Deployment" Version="1.8.3" />
    <PackageReference Include="OrchardCore.DisplayManagement" Version="1.8.3" />
    <PackageReference Include="OrchardCore.DisplayManagement.Abstractions" Version="1.8.3" />
    <PackageReference Include="OrchardCore.Html" Version="1.8.3" />
    <PackageReference Include="OrchardCore.ResourceManagement" Version="1.8.3" />
    <PackageReference Include="OrchardCore.Module.Targets" Version="1.8.3" />
    <PackageReference Include="OrchardCore.Workflows" Version="1.8.3" />
    <PackageReference Include="OrchardCore.Workflows.Abstractions" Version="1.8.3" />
  </ItemGroup>
```

What else am I missing? Is it not enough to reference both the Workflows module and the Abstractions package?

**Comment by hishamco** [comment, marked as answer]
AFAIK `OC.Json` should be a part of `2.0.0`, @MikeAlhayek @sarahelsaig correct me if I'm wrong

**Reply by MikeAlhayek** [reply]
That is correct. That is part of 2.0. Either continue to use NewtonSoft.Json or upgrade to 2.0

**Reply by weirdyang** [reply]
Ah thanks, used wrong branch to check.
